pkg/agent/flowexporter/connections/conntrack_ovs.go

```diff
@@ -201,3 +209,15 @@ func lookupProtocolMap(name string) (uint8, error) {
 	}
 	return proto, nil
 }
+
+func getMaxConnectionsInConnTrack(ovsctlClient ovsctl.OVSCtlClient) (int, error) {
```

Comment:
if the function needs `ovsctlClient`, it makes more sense to be a method of `connTrackOvsCtl`, it's not really different from `ovsAppctlDumpConnections` which also needs `ovsctlClient` and parses the output.